doc/source/contributing_code.txt

```diff
@@ -145,8 +148,17 @@ hours).
 tests passing, and no warnings, the PR is merged into master by Jim
 (???)
 
+<li>Commit comments are squished, to come comment per merged PR.
```

Comment:
"to come comment"?   When a branch is part of a PR, the commits in it should already be cleaned up and ready for public consumption.   The merge to develop/master should be non-fast-forward.